_config.yml

```diff
@@ -2,13 +2,62 @@
 domain: services.github.com
 author: GitHub, Inc.
 email: services@github.com
-title: On Demand Training
+
+
+# Site Settings
+locale                   : "en-US"
+# Keep original, do not replace
+title                    : "On Demand Training"
+title_separator          : "-"
+name                     : "GitHub, Inc."
+description              : "You're in control. We're with you along the way."
+url                      : "https://github.github.com" # the base hostname & protocol for your site e.g. "https://mmistakes.github.io"
```

Comment:
@hectorsector is this still right? Should it be github.github.com? or services.github.com?